examples/src/main/python/ml/vector_slicer_example.py

```diff
@@ -20,15 +20,18 @@
 # $example on$
 from pyspark.ml.feature import VectorSlicer
 from pyspark.ml.linalg import Vectors
-from pyspark.sql.types import Row
 # $example off$
 from pyspark.sql import SparkSession
+# $example on$
```

Comment:
the SparkSession is usually hiden in other examples. I can add it you really want